src/etl/construct_etl.py

```diff
@@ -134,168 +121,196 @@ def _process_sub_type(self, sub_type):
 
     def get_generators(self, construct_data, data_provider, batch_size):
+        '''Create Generators'''
 
-        dataProviders = []
+        data_providers = []
         release = ""
         constructs = []
         construct_synonyms = []
-        construct_secondaryIds = []
-        crossReferenceList = []
-        componentDetails = []
-        componentNoGeneDetails = []
-        nonBgiComponents = []
+        construct_secondary_ids = []
+        cross_reference_list = []
+        component_details = []
+        component_no_gene_details = []
+        non_bgi_components = []
 
         counter = 0
-        dateProduced = construct_data['metaData']['dateProduced']
+        date_produced = construct_data['metaData']['dateProduced']
 
-        dataProviderObject = construct_data['metaData']['dataProvider']
+        data_provider_object = construct_data['metaData']['dataProvider']
 
-        dataProviderCrossRef = dataProviderObject.get('crossReference')
-        dataProvider = dataProviderCrossRef.get('id')
-        dataProviderPages = dataProviderCrossRef.get('pages')
-        dataProviderCrossRefSet = []
+        data_provider_cross_ref = data_provider_object.get('crossReference')
+        data_provider = data_provider_cross_ref.get('id')
+        data_provider_pages = data_provider_cross_ref.get('pages')
+        data_provider_cross_ref_set = []
 
-        loadKey = dateProduced + dataProvider + "_construct"
+        load_key = date_produced + data_provider + "_construct"
 
         # TODO: get SGD to fix their files.
 
-        if dataProviderPages is not None:
-            for dataProviderPage in dataProviderPages:
-                crossRefCompleteUrl = ETLHelper.get_page_complete_url(dataProvider, self.xrefUrlMap, dataProvider,
-                                                                      dataProviderPage)
-
-                dataProviderCrossRefSet.append(ETLHelper.get_xref_dict(dataProvider, dataProvider, dataProviderPage,
-                                                                       dataProviderPage, dataProvider,
-                                                                       crossRefCompleteUrl,
-                                                                       dataProvider + dataProviderPage))
-
-                dataProviders.append(dataProvider)
-                logger.info("data provider: " + dataProvider)
+        if data_provider_pages is not None:
+            for data_provider_page in data_provider_pages:
+                cross_ref_complete_url = ETLHelper.get_page_complete_url(data_provider,
+                                                                         self.xref_url_map,
+                                                                         data_provider,
+                                                                         data_provider_page)
+
+                data_provider_cross_ref_set.append(ETLHelper.get_xref_dict(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \.